package com.urmanco.openweathermap.data.source.remote

     const val BASE_URL = "https://api.openweathermap.org/data/2.5/"
     const val BASE_URL_IMAGES = "http://openweathermap.org/img/wn/"
     const val NETWORK_ERROR = "Network Error"
     const val GENERIC_ERROR = "Generic Error"
     const val LAT = "37.55"
     const val LON = "45.08"
     const val APP_ID = "e8ec0c1db88e57d94ed46278fb4284b0"
     const val UNITS = "metric"
     const val UNKNOWN_EXCEPTION =  "unknown exception API Call"
